zephyr-mcumgr/src/bootloader.rs

```diff
@@ -0,0 +1,42 @@
+use serde::Serialize;
+
+/// Information about the bootloader
+#[derive(Serialize)]
+pub enum BootloaderInfo {
+    /// MCUboot bootloader
+    MCUboot {
+        /// Bootloader mode
+        ///
+        /// See [TODO] for more information.
+        mode: i32,
+        /// Bootloader has downgrade prevention enabled
+        no_downgrade: bool,
+    },
+    /// Other bootloader
+    Other {
+        /// Name of the bootloader
+        name: String,
+    },
+}
```

Comment:
It isn't exposed to python. The function uses serde-pyobject to convert it to PyAny.